### PR TITLE
- Remove unused context from homepage view

### PR DESCRIPTION
**Pull request by ahmed-faraz46**
- Moved get_featured_courses function to helpers.py

### Description

[LP-2090](https://philanthropyu.atlassian.net/browse/LP-2090)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/1283)

### CODE REVIEW

File: lms/djangoapps/homepage/helpers.py [hunk 1]
```diff
@@ -1,25 +1,7 @@
...
-    context = {
-        'courses': featured_courses,
-        'categories': featured_categories if status_code == 200 else [],
-    }
-
-    return render_to_response("homepage/homepage.html", context)
+from xmodule.modulestore.django import modulestore
 
 
 def get_featured_courses(user):
```
Comment (saad-waqar) [inline lms/djangoapps/homepage/helpers.py:7]: lets check if this method is not being used anywhere else. If it is not, lets remove it!

Comment (saad-waqar) [reply lms/djangoapps/homepage/helpers.py:7]: Also lets remove featured categories from nodebb side and edx Nodebb client

Comment (ahmed-faraz46) [reply lms/djangoapps/homepage/helpers.py:7]: @saad-waqar actually @emraan004 suggested to move it to helpers for future uses and due to same reason I didn't remove category method.

Comment (saad-waqar) [reply lms/djangoapps/homepage/helpers.py:7]: Understood! Lets keep this code in case it is required in future.

**Pull request merged**